dep/mysql/CMakeLists.txt

```diff
@@ -32,5 +32,6 @@ else()
 endif()
 
 target_include_directories(mysql SYSTEM INTERFACE "${MYSQL_INCLUDE_DIR}")
-target_link_libraries(mysql INTERFACE "${MYSQL_LIB_DIR}/libmysql.lib")
-target_compile_definitions(mysql INTERFACE _ALLOW_ITERATOR_DEBUG_LEVEL_MISMATCH)
\ No newline at end of file
+# target_link_libraries(mysql INTERFACE "${MYSQL_LIB_DIR}/libmysql.lib")
+target_link_libraries(mysql INTERFACE "${MYSQL_LIB_DIR}/libmysqlclient.18.dylib")
```

Comment:
I guess we should wrap this in an `IF(MACOS)`. It would be nicer if we could tell cmake to locate this for us, but I'm not sure if that's possible?

Reply:
I split this up into two ifs, we might need a different solution if .dylib isn't universal for unix